utils/cockpituous-release

```diff
@@ -16,13 +16,10 @@ job release-source
 job release-srpm -V
 
 # Do fedora builds for the tag, using tarball
-job release-koji master
```

Comment:
I think you want master (that is rawhide)

Reply:
Yes, definitively.

Reply:
Fixed now.